Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cql3-parser"
-version = "0.4.1"
+version = "0.4.2"
 authors = ["Claude <claude.warren@instaclustr.com>"]
```

Comment:
why do we have this name and email?

Reply:
Claude was the employee who originally wrote this code.